test/calibration/test_calibration_basics.py

```diff
@@ -0,0 +1,25 @@
+# pylint: disable=import-outside-toplevel
```

Comment:
Would it be better to just disable this for the entire `test` dir?

Reply:
Good point. I'll try to do this.

Reply:
Not sure if there's a way to do this without introducing another .pylintrc file in addition to ".pylintrc" and ".pylintrc-packaged-files". Worth it?

Reply:
Wouldn't `.pylintrc` be a good candidate for implementation details? Maybe a better separation than what we have is:

* `.pylintrc` -> `.pylintrc-implementation-details`
* `.pylintrc-packaged-files` -> `.pylintrc-public-code` (Not liking the name, but you get the idea)

Reply:
Sure, but my point was that we would need a third pylintrc as well (since we probably want import-outside-toplevel on internal files that are not tests).

Reply:
I'll see if I find a solution that makes sense.

Reply:
Alright I've done a considerable revamp, and managed to remove a large amount of individual linter suppressions.
The Python linting part in lint.sh now looks like this:
```
# Python linting
runPylint "$publicPythonFiles" ".pylintrc" || exit $?
runPylint "$internalPythonFiles" ".pylintrc-internal" || exit $?
runPylint "$testsPythonFiles" ".pylintrc-tests" || exit $?

runFlake8 "$publicPythonFiles" ".flake8" || exit $?
runFlake8 "$internalPythonFiles" ".flake8-internal" || exit $?
runFlake8 "$testsPythonFiles" ".flake8-internal" || exit $?

runDarglint "$publicPythonFiles" || exit $?

runBlackCheck "$allPythonFiles" || exit $?
```
Notes:
- I call the cfg for public files just ".pylintrc" etc because this is the one I want the end-user's editor to pick up by default.
- Having only two cfg files for flake8 is intentional. There was no need for a third one for tests here.

Reply:
This was confusing, in otherwize neat world: `runFlake8 "$testsPythonFiles" ".flake8-internal" || exit $?`.

This looks like a copy paste error, but it could be intentional. I would make it on of:
* `runFlake8 "$testsPythonFiles" ".flake8-tests" || exit $?`
* `runFlake8 "$testsPythonFiles" ".flake8-internal-and-tests" || exit $?`

Reply:
Except for that, nice improvement!

Reply:
Fair point. I ended up doing this:
```
runFlake8 "$internalPythonFiles" ".flake8-internal" || exit $?
runFlake8 "$testsPythonFiles" ".flake8-tests" || exit $?
```
The two .flake8 files are identical at the moment, but I suspect that won't last, so now it's easily to modify just one of them. And it exactly mirrors the pylint setup.